src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -30,23 +35,38 @@ final class CheckoutContext implements Context
     /** @var AbstractBrowser */
     private $client;
 
+    /** @var ApiClientInterface */
+    private $orderClient;
```

Comment:
```suggestion
    private $ordersClient;
```
to be consistent with other contexts